src/content/en/fundamentals/primers/service-workers/the-players.markdown

```diff
@@ -0,0 +1,41 @@
+---
+layout: shared/narrow
+published: false
+title: "The Players"
+description: "A basic service worker implementation has two types of players."
+published_on: 2015-10-01
+updated_on: 2015-10-01
+order: 3
+translation_priority: 0
+authors:
+- josephmedley
+---
+
+<p class="intro">
+  A basic service worker implementation has two types of players.
+</p>
+
+ 
+### Clients
+
+These players install and use a service worker. Generally, they  are web
+pages. Though there are [other types of clients](https://developer.mozilla.org/en-US/docs/Web/API/Client)
+,  this primer focuses on web page clients.
```

Comment:
Move the `,` to the previous line, to remove the space in the rendered Markdown.

While a `SharedWorker` might be a service worker client, they can't register/install a service worker.

 I don't think it's useful to bring up the existence of non-web page clients, especially so early on in what's meant to be a beginner's primer. It's not a distinction that will be relevant for most developers.